### PR TITLE
Longest common subsequence in C++

### PR DESCRIPTION
**Pull request by Yukti-1**
### Have you read the [Contributing Guidelines on Pull Requests]
Yes
### Description
I have added the algorithm to print the longest common subsequence between two strings and also it's length.
For example - If the input strings are "QWERTYHU" and "WROPYHUI" , the output will be "WRYHU" and length is 5.
The code has been written in C++.
I have also updated the read me section under cpp.

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Solves the issue #2948

### CODE REVIEW

**Comment by Yukti-1** [comment]
Could you please review my PR @HarshCasper

**Comment by ankitaggarwal23** [comment]
@Yukti-1 Duplicate issue. Please checkout [#438](https://github.com/TesseractCoding/NeoAlgo/issues/438) .
Happy Coding!

**Comment by Yukti-1** [comment]
> @Yukti-1 Duplicate issue. Please checkout [#438](https://github.com/TesseractCoding/NeoAlgo/issues/438) .
> Happy Coding!

Actually, I have added the code which prints the subsequence which is longest common and the issue you mentioned just prints
it's length. So, I think they are not same. 
@ankitaggarwal23 Could you please look into this?

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -250,6 +250,7 @@ _add list here_
 - [Length of longest palindromic subsequence](dp/longest_palin_subseq.cpp)
 - [Minimum number of insertions and deletions](dp/min_ins_del.cpp)
 - [Unbounded Knapsack](dp/unbounded_knapsack.cpp)
+- [Print_Longest_Common_Subsequence](dp/Print_Longest_Common_Subsequence.cpp)
```
Comment (HarshCasper) [inline C-Plus-Plus/README.md]: Remove Underscores

File: C-Plus-Plus/dp/Print_Longest_Common_Subsequence.cpp [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    return s;
+}
+
+int main()
+{
+    string str_1, str_2;
+    cout << "Enter two strings: ";
+    cin >> str_1 >> str_2;
+    longest_common_sub(str_1,str_2);
+}
```
Comment (HarshCasper) [inline C-Plus-Plus/dp/Print_Longest_Common_Subsequence.cpp:94]: Add sample I/O and complexity

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/dp/Print_Longest_Common_Subsequence.cpp (hunk 2 above)
Comment (Yukti-1) [reply C-Plus-Plus/dp/Print_Longest_Common_Subsequence.cpp:94]: Added

File: C-Plus-Plus/README.md (hunk 1 above)
Comment (Yukti-1) [reply C-Plus-Plus/README.md]: Done the changes

File: C-Plus-Plus/dp/Print_Longest_Common_Subsequence.cpp [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    SAMPLE INPUT - 
+    1) Enter two strings: QWERTYHU   WROPYHUI
+       Length of the longest common subsequence is 5
+       Longest common subsequence is WRYHU
+       
+    2) Enter two strings: CODING   ENCODING
+       Length of the longest common subsequence is 6
+       Longest common subsequence is CODING
+          
+    */
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/dp/Print_Longest_Common_Subsequence.cpp]: Don't write inside the main function
- Add sample I/O at the end of the Code
- Add Time and Space Complexity at the end of the Code   
- Add the new line at the EOF

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/dp/Print_Longest_Common_Subsequence.cpp (hunk 3 above)
Comment (Yukti-1) [reply C-Plus-Plus/dp/Print_Longest_Common_Subsequence.cpp]: Done the said changes.

**Review by ankitaggarwal23 — COMMENTED** [review]
Why your some parts of the code matches from [here](https://leetcode.com/problems/shortest-common-supersequence/discuss/991904/Simple-O(nm)-solution-with-detailed-explanation-C%2B%2B)

**Comment by Yukti-1** [comment]
The code you mentioned above follows the same DP approach to solve a different problem from the one I submitted though I didn't copy from here nor did I see it before.
The resulting similarities which you found may be because in any DP problem, the general way is to construct a matrix(as far as I know) and store the result somewhere(here matrix) and use it later to avoid computing same subproblems again. 
And yeah that's it. @ankitaggarwal23

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**